dist/neo.blockchain.sync.js

```diff
@@ -94,7 +94,7 @@ function sync (blockchain) {
 
     sync.clock2 = setInterval(() => {
       if (sync.runLock) {
-        blockchain.localNode.verify()
+        blockchain.localNode.storage.verify()
```

Comment:
I'm not sure how this is supposed to be working but the eventual result is that you are pulling block 0:n from mongo. To prevent mongo from throwing a > 32mb exception I've modified this in my dev environment to and am running a sync overnight.
`
        let startIndex = blockchain.localNode.storage.index - 30
        if (startIndex < 0) {
          startIndex = 0
        }
        blockchain.localNode.storage.verify(startIndex, blockchain.localNode.storage.index)
`